cdap-common/src/main/java/co/cask/cdap/common/io/CaseInsensitiveEnumTypeAdapterFactory.java

```diff
@@ -85,11 +95,11 @@ public T read(JsonReader reader) throws IOException {
     };
   }
 
-  private String toLowercase(Object o) {
+  private static String toLowercase(Object o) {
```

Comment:
you no longer need these as separate methods, they should be in the respective function.